CHANGELOG.md

```diff
@@ -266,6 +270,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Fixed `get_metadata` calls in examples to remove `results_dir` argument
 - Removed YouTube video temporarily
 
+### Added
```

Comment:
Any new changes to the changelog must be added in the unreleased section

Reply:
Updated.